app-catalog/samples/compose/src/main/java/com/google.android.catalog.app.compose/ComposeSample.kt

```diff
@@ -23,7 +23,12 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import com.google.android.catalog.framework.annotations.Sample
 
-@Sample(name = "Compose sample", "Shows how to add a compose target in the catalog")
+@Sample(
+    name = "Compose sample",
+    description = "Shows how to add a compose target in the catalog",
+    documentation = "https://github.com/google/casa-android#create-sample-modules",
+    sourcePath = "compose/src/main/java/com/google.android.catalog.app.compose/ComposeSample.kt"
```

Comment:
Don't you think there is a risk of the connection breaking if you manually type in the source path?

If we can ensure that the file name and the name of the annotation target (function, class) are the same, it seems that SampleProcessor can infer the sourcePath.

Reply:
I don't want to restrict to it. There might be cases that the source code might be in different paths format or maybe have multiple samples entry points in one-file. But maybe I can add that if the sourcePath is empty it defaults to the file path